tools/hardhat-example/.env.example

```diff
@@ -4,3 +4,5 @@ RECEIVER_PRIVATE_KEY=0x2e1d968b041d84dd120a5860cee60cd83f9374ef527ca86996317ada3
 RELAY_ENDPOINT='http://localhost:7546'
 
 MNEMONIC="here is where your twelve words mnemonic should be put my friend"
+
+TESTNET_OPERATOR_PRIVATE_KEY='<if you want to deploy on testnet go to https://portal.hedera.com/ to setup your private key>'
```

Comment:
I like this 👍

Reply:
Althought I think this is breaking the CI

Reply:
Q: why is this needed vs just changing your `OPERATOR_PRIVATE_KEY`
The current value assumes local node per the comment but if you changed networks it makes sense to change value no?

Reply:
Yes and no. We could indeed use the same key. But having different ones allows the user to deploy to different networks at the same time without changing the configuration or the env file.